packages/financial-templates-lib/src/clients/InsuredBridgeL2Client.ts

```diff
@@ -38,7 +44,8 @@ export class InsuredBridgeL2Client {
     readonly chainId: number = 0,
     readonly startingBlockNumber: number = 0,
     readonly endingBlockNumber: number | null = null,
-    readonly fallbackL2Web3s: Web3[] = []
+    readonly fallbackL2Web3s: Web3[] = [],
+    readonly l2BlockLookback: number = 0
```

Comment:
Why can't you use `startingBlockNumber` and or `endingBlockNumber` in place of `l2BlockLookback`? Arent' these redundant?